chapter-11/README.md

```diff
@@ -1,7 +1,11 @@
-# Chapter 11: Generating Source Archives and Binary Distributions
+# Chapter 11: Packaging Projects
 
 - [Generating source archives](recipe-01/README.md)
-- [Generating binary archives](recipe-02/README.md)
-- [Generating DEB packages](recipe-03/README.md)
-- [Generating RPM packages](recipe-04/README.md)
-- [Generating an NSIS installer for Windows](recipe-05/README.md)
+- [Generating binary archives](recipe-01b/README.md)
+- [Generating DEB packages](recipe-01c/README.md)
+- [Generating RPM packages](recipe-01d/README.md)
+- [Generating an NSIS installer for Windows](recipe-01e/README.md)
+- [Distributing a C++/Python project built with CMake/pybind11 via PyPI](recipe-02/README.md)
```

Comment:
Accordingly, these will be bumped one number up: 2->3, 3->4, 4->5, 5->6

Reply:
I'll wait for Travis to finish and then merge.

Reply:
Once CI is through and you are happy, please approve but do not merge, I will then merge and want to keep it as reminder to myself to also adjust TC and internal document numbering.